cookbooks/bcpc-hadoop/recipes/hbase_master.rb

```diff
@@ -13,6 +13,66 @@
     'docopy' => true
 }
 
+# Define zabbix triggers for alarming
+# The Hbase triggers depend on the namenode triggers too. So the namenode triggers
+# are also defined here. This takes care of creation of triggers irrespective of
+# how namenode and hbase_master appear in the runlist. Assumption is that hdfs will
+# be installed if hbase is installed
+node.default['bcpc']['hadoop']['graphite']['queries']['namenode'] = {
```

Comment:
Dude; this is hbase master not namenode. Please understand the trigger dependency race you were seeing or add support in `graphite_to_zabbix.rb` to only add trigger dependencies once a trigger is created (and log that you are skipping the dependency).